openstack/swift/templates/sftp-deployment.yaml

```diff
@@ -0,0 +1,52 @@
+{{- if .Values.sftp.enabled }}
+kind: Deployment
+apiVersion: extensions/v1beta1
+
+metadata:
+  name: swift-sftp
+  labels:
+    release: "{{.Release.Name}}"
+spec:
+  revisionHistoryLimit: 5
+  strategy:
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      component: swift-sftp
+  template:
+    metadata:
+      labels:
+        component: swift-sftp
+    spec:
+      containers:
+        - name: sftp
+          image: hub.global.cloud.sap/monsoon/swift-sftp:{{ .Values.image_version_auxiliary_sftp  }}
+          args: ['server', '-f', '/etc/swift/swift-sftp.conf']
+          env:
+          {{- if $.Values.resources.enabled }}
+          resources:
+            requests:
+              cpu: "1000m"
+              memory: "512Mi"
+            limits:
+              cpu: "2000m"
+              memory: "1024Mi"
```

Comment:
This seems excessive, esp. the CPU allocation. On which measurements are these requests based?

Reply:
Over the thumb, kubectl top didn't worked for me, could you assist?